<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_ActionDrawArc2POptions</class>
 <widget class="QWidget" name="LC_ActionDrawArc2POptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="lHeight">
     <property name="toolTip">
      <string>Height of the arc (the distan—Åe between chord and top arc point)</string>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QRadioButton" name="rbPos">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Counterclockwise</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../res/extui/extui.qrc">
       <normaloff>:/extui/dirpos.png</normaloff>:/extui/dirpos.png</iconset>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lRadius">
     <property name="toolTip">
      <string>Radius of arc</string>
     </property>
     <property name="text">
      <string>Radius:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QRadioButton" name="rbNeg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clockwise</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../res/extui/extui.qrc">
       <normaloff>:/extui/dirneg.png</normaloff>:/extui/dirneg.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lAngle">
     <property name="toolTip">
      <string>Central angle of the arc</string>
     </property>
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="leValue"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lLength">
     <property name="toolTip">
      <string>Length of the arc</string>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../res/extui/extui.qrc"/>
 </resources>
 <connections/>
</ui>
